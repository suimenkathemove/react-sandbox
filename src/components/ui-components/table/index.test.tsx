import { render } from '@testing-library/react';

import { defaultProps } from './index.stories';

import 'jest-styled-components';
import { Table } from './';

const setup = () => render(<Table {...defaultProps} />);

describe('Table', () => {
  test('render', () => {
    const { asFragment } = setup();
    expect(asFragment()).toMatchSnapshot();
  });
});
