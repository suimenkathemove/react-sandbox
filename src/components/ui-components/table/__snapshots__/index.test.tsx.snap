// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table render 1`] = `
<DocumentFragment>
  .c0 {
  display: table;
  border-collapse: separate;
  background-color: white;
}

.c1 {
  display: table-header-group;
}

.c2 {
  display: table-row;
}

.c3 {
  position: -webkit-sticky;
  position: sticky;
  top: 0;
  z-index: 1;
  display: table-cell;
  background-color: white;
  border-bottom: 1px solid black;
}

.c4 {
  display: table-row-group;
}

.c5 {
  display: table-cell;
}

<table
    class="c0"
  >
    <thead
      class="c1"
    >
      <tr
        class="c2"
      >
        <th
          class="c3"
        >
          name
        </th>
      </tr>
    </thead>
    <tbody
      class="c4"
    >
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          0
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          1
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          2
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          3
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          4
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          5
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          6
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          7
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          8
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          9
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          10
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          11
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          12
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          13
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          14
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          15
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          16
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          17
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          18
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          19
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          20
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          21
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          22
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          23
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          24
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          25
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          26
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          27
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          28
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          29
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          30
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          31
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          32
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          33
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          34
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          35
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          36
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          37
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          38
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          39
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          40
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          41
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          42
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          43
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          44
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          45
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          46
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          47
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          48
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          49
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          50
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          51
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          52
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          53
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          54
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          55
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          56
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          57
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          58
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          59
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          60
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          61
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          62
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          63
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          64
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          65
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          66
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          67
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          68
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          69
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          70
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          71
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          72
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          73
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          74
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          75
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          76
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          77
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          78
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          79
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          80
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          81
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          82
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          83
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          84
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          85
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          86
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          87
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          88
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          89
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          90
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          91
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          92
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          93
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          94
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          95
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          96
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          97
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          98
        </td>
      </tr>
      <tr
        class="c2"
      >
        <td
          class="c5"
        >
          99
        </td>
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
